brigade/plugins/inventory/ansible.py

```diff
@@ -0,0 +1,179 @@
+import configparser
```

Comment:
Something weird is going on here. In Python 2.7 this module is called `ConfigParser` it was renamed to `configparser` in Python 3. I don't understand why this would work and how come Tox doesn't pick it up. Unless I'm missing something obvious there's a bug somewhere.

Reply:
yeah, that's weird...